cmd/lotus-miner/pieces.go

```diff
@@ -61,9 +68,54 @@ var piecesListCidInfosCmd = &cli.Command{
 			return err
 		}
 
+		w := tablewriter.New(tablewriter.Col("CID"),
```

Comment:
Isn't this gonna get printed in non verbose mode too

Reply:
Nope, tablewriter doesn't print anything with no entries

Reply:
oh nice